proposals/semi-auto-properties.md

```diff
@@ -54,7 +54,7 @@ The following changes are to be made to [classes.md](https://github.com/dotnet/c
 
 ```diff
 - If the auto-property has no set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
-+ If the auto-property has semicolon-only get accessor without a set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the backing field of the property.
++ If the auto-property has semicolon-only get accessor (without a set accessor or with an init-only accessor), the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the backing field of the property.
 ```
```

Comment:
lol.  i got so confused fro a second.  i was going to say these two seem almost identical and should be merged.  then i realized i was looking at a diff 🗡️

Reply:
Yeah reviewing a diff in a diff hurts eyes 😄

Reply:
hrmm... does teh constructor actually assign to the field?  it doesn't go through the init-setter?

Reply:
@CyrusNajmabadi I think the spec is now wrong regarding this point. I focused more on the first sentence. We need to adjust.

Reply:
yeah, i think the second sentence needs to be clear that it's only covering the `{get;}` case and not the `init` case.

Reply:
Actually it seems like it's accurate already? It mentions `getter-only auto property`, so it doesn't apply for the "with an init-only accessor" part of the previous sentence.

Reply:
right.  i think the language is correct, but potentially confusing in context.  i wasn't sure if `getter only` meant: only `{ get; }`, or also somehow applied to get+init (since hte prior sentence covers that).  I feel like it could use with jsut a little clarity, only to avoid confusion.